src/test/ui/closure_context/issue-26046-fn-mut.stderr

```diff
@@ -6,6 +6,9 @@ error[E0525]: expected a closure that implements the `Fn` trait, but this closur
 15 | |         num += 1;
 16 | |     };
    | |_____^
+17 | 
```

Comment:
cc @estebank -- @tommyip mentioned that there is a trailing space after this `17 | ` -- this doesn't bother me per se, but is that just how the diagnostics output works in general?